modules/administration-guide/pages/limiting-the-number-of-workspaces-that-a-user-can-run-simultaneously.adoc

```diff
@@ -0,0 +1,41 @@
+:_content-type: PROCEDURE
+:navtitle: Limiting the number of workspaces that a user can run simultaneously
+:description: Limiting the number of workspaces that a user can run simultaneously
+:keywords: administration guide, number, workspaces
+
+[id="limiting-the-number-of-workspaces-that-a-user-can-run-simultaneously_{context}"]
+= Limiting the number of workspaces that a user can run simultaneously
+
+{prod-short} enables users to run multiple workspaces with different configurations simultaneously.
+
+You can limit the number of workspaces that a user can run simultaneously.
```

Comment:
If you refactor the first topic, make similar changes to this one, too.

Reply:
Thank you.